R/get_onetable.R

```diff
@@ -30,10 +30,12 @@
 #' usethis::use_data(onetable, overwrite = TRUE)
 #' }
 #'
-get_onetable <- function(usaid_metadata_file, vintage = 2021, country_geometries = country_coords) {
+get_onetable <- function(usaid_metadata_file = NULL, vintage = 2021, country_geometries = country_coords) {
 
   ## Country List
   # From COVID sources.
+  # TODO: This is a complete bodge and needs to be fixed when #20 is pulled in
```

Comment:
We shouldn't be using get_covid_df() to pull in just metadata, but I've changed this significantly in #20, so I need to work up a solution there first.